src/js/tabs/pid_tuning.js

```diff
@@ -64,6 +64,8 @@ TABS.pid_tuning.initialize = function (callback) {
         $('#content').load("./tabs/pid_tuning.html", process_html);
     }
 
+    const vbatpidcompensationIsUsed = (semver.gte(CONFIG.apiVersion, "1.16.0")) && (semver.lt(CONFIG.apiVersion, API_VERSION_1_44));
```

Comment:
Too much parenthesis again :P
```
const vbatpidcompensationIsUsed = (semver.gte(CONFIG.apiVersion, "1.16.0") && semver.lt(CONFIG.apiVersion, API_VERSION_1_44));
```
or simply:
```
const vbatpidcompensationIsUsed = semver.gte(CONFIG.apiVersion, "1.16.0") && semver.lt(CONFIG.apiVersion, API_VERSION_1_44);
```

maybe we need to add a method to `semver` that allows two versions :)

Reply:
@McGiverGim  the idea to add method to semver to allow between 2 API versions sounds good